### ISSUE TITLE
The app crashes after switching between Direct3D 11 and Direct3D 12

### ISSUE THREAD

**Issue by antoniodesousa** [TheRhysWyrill/XBSX2#4]
## Description

Like the title says, when switching between Direct3D 11 and Direct3D 12, the app becomes unresponsive and I have to quit and restart the app again. It does keep the changes though. It just quite annoying that I have to restart the app when I want to use a specific version of Direct3D in some games, like Gran Turismo 3 and 4 to fix the reflection in the cars when using Direct3D 12.

### Expected behavior

The app doesn't become unresponsive when switching between Direct3D 11 and 12.

### Actual behavior

The app becomes unresponsive when switching between Direct3D 11 and 12.

### Steps to reproduce the bug

1. Go to settings 
2. Select Display Settings menu
3. Change Renderer to Direct3D 11 or 12

### Console

Xbox Series S

**Comment by TheRhysWyrill** [comment]
Unfortunately this seems to be an issue that I can't replicate, does it still happen with the latest release?

**Comment by Bl4ckH4nd87** [comment]
I tested this a few times today, doesn't happen on my end at all. You probably messed something up in your settings. I test every update before it goes live and then I test it more, just in case. I have never come across this issue. Sometimes the app might go back to the main menu when switching the renderer, but there isn't a crash. I'd recommend deleting your pcsx2.ini so the app can generate a fresh one, and don't mess with settings you do not understand.

**Comment by antoniodesousa** [comment]
I reinstalled the app again today. I started up. I didn't change ANY settings. All settings were the default. I changed the rendering to Direct3D 11. The screen became black and after awhile the xbox dashboard was shown. It happens every time that I change the rendering to something else. Not just between Direct3D drivers.

And I have been a PCSX2 user for many years, so I'm pretty sure how to configure the emulator. Sadly, there aren't any error logs that might indicate what's going wrong.
